marketpulse/main/templates/fxosprice_view.html

```diff
@@ -68,6 +68,12 @@ <h3 class="title">Contribution</h3>
         {% endfor %}
       </div>
     </div>
+    <div class="col-md-6">
+      <div class="contribution-label">
+        <legend>Comments</legend>
+        <div> {{ contribution.comment }} </div>
+      </div>
+    </div>
```

Comment:
You don't need to create another div with the `col-md-6` div. Just include comments inside the one above.